Cargo.toml

```diff
@@ -8,3 +8,126 @@ members = [
   "runtime/mainnet",
   "runtime/devnet",
 ]
+
+[workspace.package]
+authors = ["Anonymous"]
+edition = "2021"
+homepage = "https://substrate.io"
+license = "Unlicense"
+repository = "https://github.com/paritytech/extended-parachain-template/"
+
+[workspace.dependencies]
+
+# Common
+clap = {version = "4.3.24", features = ["derive"]}
+color-print = "0.3.4"
+futures = "0.3.28"
+hex-literal = {version = "0.4.1"}
+jsonrpsee = {version = "0.16.2", features = ["server"]}
+log = {version = "0.4.20", default-features = false }
+parity-scale-codec = {version = "3.2.2", default-features = false, features = ["derive"]}
```

Comment:
We can still use the `codec` alias here. This will work: 
```toml
# In root level workspace `Cargo.toml`
codec = { package = "parity-scale-codec", version = "3.2.2", default-features = false, features = [
  "derive",
] }
```
and then say in the workspace member where you're using it, say `pallet-motion/Cargo.toml` you can use : 
```toml
# pallet-motion
codec = { package = "parity-scale-codec", workspace = true, default-features = false, features = ["derive"] }
# ... snip --
[features]
std = [ 
 "codec/std",
 # .. snip --
]
```

Reply:
This works, but generates warnings `unused manifest key: dependencies.codec.package`. 

Using without the additional package = "parity-scale-codec" results in compiler error:
```
error: Could not find `parity-scale-codec` in `dependencies` or `dev-dependencies`
note: this error originates in the derive macro `$crate::codec::Encode` which comes from the expansion of the macro `impl_opaque_keys`
```

Reply:
Yes I noticed this as well. It seems cargo cannot figure out where the "key" is being made use of as it's hidden behind macro expansions. Perhaps the best option is that we get rid of the aliases (revert commit [3a8b713](https://github.com/paritytech/extended-parachain-template/pull/68/commits/3a8b713cfc1bc0e706ffe0a577b7d9852c42712b)) , the alternative being to make an exception for just codec for each crate using it
```toml
codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = ["derive"] }
```
I think it's much cleaner to just revert the commit rather than maintain 3 separate copies of the same dependency. Let's do it your way @evilrobot-01 :)